textproto/header.go

```diff
@@ -309,6 +309,90 @@ func isSpace(c byte) bool {
 	return c == ' ' || c == '\t'
 }
 
+func validHeaderFieldByte(b byte) bool {
```

Comment:
A "header field" is the whole line (key + value). Can we rename this to `validHeaderKeyByte`?

Reply:
Makes sense!